docs-client/src/containers/HomePage/index.tsx

```diff
@@ -13,12 +13,71 @@
  * License for the specific language governing permissions and limitations
  * under the License.
  */
-
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import Tooltip from '@material-ui/core/Tooltip';
 import Typography from '@material-ui/core/Typography';
+import dayjs from 'dayjs';
+import relativeTime from 'dayjs/plugin/relativeTime';
 import React from 'react';
+import Section from '../../components/Section';
+import { convertLongToUTCDate, Versions } from '../../lib/versions';
+
+interface OwnProps {
+  versions?: Versions;
+}
 
-export default class HomePage extends React.PureComponent {
+dayjs.extend(relativeTime);
+
+function formatReadableTime(dateTime: number) {
```

Comment:
nit: maybe OK to inline this method because it's used only in a single place?

Reply:
I think this method offer description and I will add a condition.